frontend/simple/views/components/GroupsMinIncome.vue

```diff
@@ -1,24 +1,60 @@
 <template>
-  <section class="has-text-right">
-    <p class="min-income-label has-text-grey">Min Income</p>
-    <p class="min-income title is-1 is-marginless"
-      data-test="minIncome">
-      {{ currency }}{{ group.incomeProvided }}
-    </p>
-    <a href="#">Propose change</a>
-  </section>
+  <div>
+    <div class="has-text-right">
+      <i18n class="min-income-label has-text-grey">Min Income</i18n>
+      <p class="min-income title is-1 is-marginless"
+        data-test="minIncome">
+        {{ currency }}{{ group.incomeProvided }}
+      </p>
+      <a href="#" @click.prevent="toggleModal">
+        <i18n>Propose change</i18n>
+      </a>
+    </div>
+    <modal-form
+      :isActive="isActive"
+      :title="modalTitle"
+      :subTitle="modalSubTitle"
+      :submitText="modalSubmitText"
+      :onSubmit="handleSubmit"
+      :onClose="toggleModal"
+    >
+      <p class="has-text-centered">A step to change mincome is on its way!</p>
+    </modal-form>
```

Comment:
Ah, indeed, here is the same code again. So yeah, I recommend putting this modal once in a higher-level view, and then using either SBP events or an SBP selector to make it visible. :)

Reply:
Same question as before :)